docs/rst/user_guide/data/radprops.rst

```diff
@@ -3,17 +3,15 @@
 Atmosphere radiative properties
 ===============================
 
-.. warning:: This content is outdated.
-
 An atmosphere radiative properties data set provide the collision (absorption,
 extinction, scattering) coefficient and albedo values within the atmosphere.
 
 Data sets access
 ----------------
 
 Atmosphere radiative properties data sets are created by
-:class:`~eradiate.radprops.rad_profile.RadProfile` objects'
-:meth:`~eradiate.radprops.rad_profile.RadProfile.to_dataset` method.
+:class:`~eradiate.radprops._core.RadProfile` objects'
+:meth:`~eradiate.radprops._core.RadProfile.eval_dataset` method.
```

Comment:
I'd expect that not to render correctly: `RadProfile` is referenced as `eradiate.radprops.RadProfile`. Anyway, no need to bother, I think the following should work:
```suggestion
:meth:`.RadProfile.eval_dataset` method.
```